x-pack/plugins/observability/public/hooks/slo/use_create_slo.tsx

```diff
@@ -55,7 +55,7 @@ export function useCreateSlo() {
               <RedirectAppLinks coreStart={services} data-test-subj="observabilityMainContainer">
                 <FormattedMessage
                   id="xpack.observability.slo.create.successNotification"
-                  defaultMessage="Successfully created {name}.\n {editSLO}"
+                  defaultMessage="Successfully created {name}. {editSLO}"
```

Comment:
@shahzad31 I was getting this error. So I removed `\n`. Feel free to update, if you have a better solution

```
Error: Couldn't parse default message ("xpack.observability.slo.create.successNotification"):
--
  | Expected "\\#", "\\\\", "\\u", "\\{", "\\}", "{", [^{}\\\0-\x1F\x7f \t\n\r], end of input or whitespace but "\\" found.:
  | Successfully created {name}.\n {editSLO}

```